src/controllers/user.controller.js

```diff
@@ -0,0 +1,88 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../models/user.js';
+import { emailService } from '../services/email.service.js';
+import { userService } from '../services/user.service.js';
+import bcrypt from 'bcrypt';
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(userService.normalize));
+};
+
+const getProfile = async (req, res) => {
+  const user = await User.findByPk(req.user.id);
+
+  res.send(user);
+};
+
+const changeName = async (req, res) => {
+  const { newName } = req.body;
+
+  if (!newName) {
+    throw res.send(ApiError.badRequest('New name is required'));
```

Comment:
The `throw res.send(ApiError.badRequest('New name is required'));` line is incorrect. You should throw the `ApiError` directly and handle the response separately. Use `throw ApiError.badRequest('New name is required');` instead.